scipy/optimize/tests/test_optimize.py

```diff
@@ -223,6 +225,33 @@ def test_powell(self):
         assert_(self.funccalls <= 116 + 20, self.funccalls)
         assert_(self.gradcalls == 0, self.gradcalls)
 
+
+    @pytest.mark.xfail(reason="This part of test_powell fails on some "
+                       "platforms, but the solution returned by powell is "
+                       "still valid.")
+    def test_powell_gh14014(self):
```

Comment:
I must be missing something obvious--there is no `xfail` on the original test that was failing, just on a new test that you've added? I guess I'll probably learn something--is this somehow intercepting the previous failures in `test_powell()` proper?

Reply:
I think the git diff makes this confusing. I just moved the failing part of the test into the new function. It looks new because I had to copy-paste some of the code from the test above.

Reply:
Hmm, I'm not seeing the removed part of the test.

Reply:
Oh, I guess I see what you mean

Reply:
A bit orthogonal, but if the test can be split into independent pieces like this it is perhaps useful to have it split off to a separate test in any case (i.e., remove the xfail when fixed instead of fusing back to one test?).

Reply:
They're not really independent, though. I copy-pasted the part that calls powell, so this does the same calculation as the original `test_powell` again.